<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>File transfer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="myName">
          <property name="placeholderText">
           <string>device name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refreshPeersButton">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="peerList"/>
      </item>
      <item>
       <widget class="QCheckBox" name="enableEncryption">
        <property name="text">
         <string>Enable encryption</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>25</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
